### DISCUSSION TITLE
is anything functional yet

### DISCUSSION THREAD

**Discussion by jeezy101** [onimp/oni_multiplayer#2] [Q&A] [answered]

**Comment by zuev93** [comment, marked as answer]
Hi,
Thanks for the interest in the mod!

Currently it is still very early from calling its playable even as a WIP mod.

You can test this out and you should be able to connect/invite friend and some basics actions will be synced. Which will give impression of the multiplayer working. However your games will be desynced quite fast (in matter of seconds) and longer you play more singificant difference will occur.

Currently we are working on experimenting and thinking of the best way to sync games to allow good latency while not causing any noticable artefacts on the client side.

I'll keep updating README and releases as soon as I have more things to share.

Best regards, zuev93 :)
